ludwig/datasets/configs/ae_price_prediction.yaml

```diff
@@ -12,3 +12,4 @@ description: |
   Innerwear Data from Victoria's Secret and Others
   600,000+ innerwear product data extracted from popular retail sites
   https://www.kaggle.com/PromptCloudHQ/innerwear-data-from-victorias-secret-and-others
+task: Regression, Classification
```

Comment:
IMO lets use yaml list instead of comma separated list for consistency
```
task(s):
  - Regression
  - Classification
```

Reply:
Good look. Done!